scripts/pi-hole/php/header.php

```diff
@@ -375,21 +375,19 @@ function pidofFTL()
                         echo '"></i> Memory usage:&nbsp;&nbsp; N/A</span>';
                     }
                     ?>
-                    <br/>
-                    <?php
-                    $tempcolor = "text-vivid-blue";
-                    if (isset($temperaturelimit) && $celsius > $temperaturelimit) {
-                        $tempcolor = "text-red";
-                    }
-                    echo '<span id="temperature"><i class="fa fa-w fa-fire '.$tempcolor.'" style="width: 1em !important"></i> ';
-
+                    <!-- Only show temp info if any data is available -->
                     if ($celsius >= -273.15) {
-                        echo 'Temp:&nbsp;<span id="rawtemp" hidden>' .$celsius. '</span>';
-                    } else {
-                        echo "No temp sensor found";
-                    }
-                    echo '<span id="tempdisplay"></span></span>';
+                      <br/>
+                      <?php
```

Comment:
The `if` statement is on HTML. It is outside the PHP tags `<?php` and `?>`. 

I think the whole block should be:

```php
                    <br>
                    <!-- Only show temp info if any data is available -->
                    <?php

                    if ($celsius >= -273.15) {
                      $tempcolor = "text-vivid-blue";
                      if (isset($temperaturelimit) && $celsius > $temperaturelimit) {
                          $tempcolor = "text-red";
                      }
                      echo '<span id="temperature"><i class="fa fa-w fa-fire '.$tempcolor.'" style="width: 1em !important"></i> ';
                      echo 'Temp:&nbsp;<span id="rawtemp" hidden>' .$celsius. '</span>';
                      echo '<span id="tempdisplay"></span></span>';
                    }
                    ?>                    
```